src/shoppinglist.cpp

```diff
@@ -0,0 +1,173 @@
+#include "shoppinglist.h"
+
+///Number of properties associated with each shopping list item (price, name, amount)
+const int NUM_SHOPPINGLIST_PROPERTIES = 3;
+///Column index for name in shopping list model
+#define ITEM_NAME_COL 0
+///Column index for price in shopping list model
+#define ITEM_PRICE_COL 1
+///Column index for amount in shopping list model
+#define ITEM_AMOUNT_COL 2
```

Comment:
Lets say I want to purchase 3 pieces of generic item and that each generic item costs 10. Would it make sense for the display to also contain information on the total price?

e.g.
Generic item - 3 x - Generic Price (10) - Total price (30)